packages/api/src/car.js

```diff
@@ -64,8 +64,8 @@ export async function carGet (request, env, ctx) {
   } = request
   // gateway does not support `carversion` yet.
   // using it now means we can skip the cache if it is supported in the future
-  const url = new URL(`/api/v0/dag/export?arg=${cid}&carversion=1`, GATEWAY)
-  res = await fetch(url, { method: 'POST' })
+  const url = new URL(`/api/v0/dag/export?arg=${cid}&carversion=1`, env.GATEWAY_URL)
+  res = await fetch(url.toString(), { method: 'POST' })
```

Comment:
Do we need this change?

Reply:
VSCode intellisense was screaming at me the first param type was wrong. 
According to `lib.dom.d.ts` used by VSCide that should be of `type RequestInfo = Request | string;`
But looking at the MDN doc the first param can be 

- A string or any other object with a [stringifier](https://developer.mozilla.org/en-US/docs/Glossary/Stringifier) — including a [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) object — that provides the URL of the resource you want to fetch.
- A [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object.

so the answer is no, it's not.
Happy to revert 👍

Reply:
ok, let's keep. Just wanted to know why